pkg/estimator/client/accurate.go

```diff
@@ -32,23 +32,41 @@ import (
 
 // RegisterSchedulerEstimator will register a SchedulerEstimator.
 func RegisterSchedulerEstimator(se *SchedulerEstimator) {
-	replicaEstimators["scheduler-estimator"] = se
-	unschedulableReplicaEstimators["scheduler-estimator"] = se
+	replicaEstimators = append(replicaEstimators, se)
```

Comment:
I suggest check if "scheduler-estimator" exists in slice.

As a result, we don't need to add `UnRegisterSchedulerEstimator` any more because I see it is only used in test files which could be redundant.

Reply:
I actually think there's a flaw here too

Reply:
> we don't need to add UnRegisterSchedulerEstimator any more because I see it is only used in test files which could be redundant

I also think there's a flaw here

Reply:
> I suggest check if "scheduler-estimator" exists in slice.

But, how can I judge "scheduler-estimator" exists in slice, this may requires that `type ReplicaEstimator interface` add a `GetName()` function.

Reply:
Or, we do not judge "scheduler-estimator" exists in slice, we just:

```go
var once sync.Once

// RegisterSchedulerEstimator will register a SchedulerEstimator.
func RegisterSchedulerEstimator(se *SchedulerEstimator) {
	once.Do(func() {
		replicaEstimators = append(replicaEstimators, se)
		unschedulableReplicaEstimators = append(unschedulableReplicaEstimators, se)
	})
}
```

Reply:
done, I midified the type of `replicaEstimators`, and the redundant `UnRegisterSchedulerEstimator` function is removed.